mandatory/2-function-creation.js

```diff
@@ -15,14 +22,26 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+  if ((typeof num === "number" && num%2 === 0) && (num <= 100)) {
+    return true;
+  } else {
+    return false;
+}
+
+}
 
 /* 
 Write a function that returns a copy of the given array arr, but with the element at the given index, index removed.
 The function must NOT change the original array, arr.
 */
 
 function remove(arr, index) {
+  let newArr = [];
+  for (let i=0; i<arr.length; i++){
```

Comment:
we can also do this let newArr = [...arr]; Can we do this without creating a new array? Try using a filter! https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter